pkg/web_css/test/variables_test.dart

```diff
@@ -75,5 +75,28 @@ void main() {
 
       expect(unused, isEmpty);
     });
+
+    test('all variables used have definition', () async {
```

Comment:
Are we sure we want this?

Having an unused variable is pretty harmless is it not?

-----

If we want lints for our SASS files why not pick a dedicated tool for the purpose:
https://stylelint.io/

We can run it with extremely limited set of lints on github actions, or enable it in vscode and install stylelint in our development setups.

If we really want to do this, then I suggest we make the error more clearly document what is wrong and where.
Ideally, use something like `source_span` so that we can print nice messages.